kmod/src/format.h

```diff
@@ -897,7 +897,6 @@ struct scoutfs_net_lock {
 
 struct scoutfs_net_lock_grant_response {
```

Comment:
Now that the roots are removed this _grant_response struct is a meaningless wrapper around _net_lock.  Everywhere that the _grant_response struct is used can be replaced with _net_lock.  You can see this in the function prototypes in client.h and server.h for {client,server}_{request,response}.   The grant was an outlier only because it included the roots.  With those gone, it can go back to matching the other three core lock messages that cross the wire by sending the _net_lock struct.  That's what I was trying to describe with the first bullet in the initial "keep going" review comment.

Reply:
cool, will do.